rust/src/cliwrap.rs

```diff
@@ -61,10 +61,13 @@ pub fn entrypoint(args: &[&str]) -> Result<()> {
     let args = &args[1..];
 
     let host_type = crate::get_system_host_type()?;
+    let skip_wrap = std::env::var("RPMOSTREE_CLIWRAP_SKIP").unwrap_or("none".to_string());
+
     if matches!(
         host_type,
         SystemHostType::OstreeHost | SystemHostType::OstreeContainer
-    ) {
+    ) && !skip_wrap.to_lowercase().eq("true")
```

Comment:
I'm ok with this here but I wonder if it would read more clearly if we moved this case up very early on; there's just a minor cost of duplicating the function call.  Like doing this above (we can condense the envvar fetch too) like:

```
    if std::env::var("RPMOSTREE_CLIWRAP_SKIP").is_some() {
        return cliutil::exec_real_binary(name, args);
    }
```

or so?